classes/handler/APIHandler.php

```diff
@@ -190,14 +196,21 @@ public function getApp()
      * Compiles the URI path pattern from the context, api version and the
      * unique string for the this handler.
      *
+     * @param  bool  $contextualApi
```

Comment:
> Can you explain this parameter in the docblock?

@NateWr  Before that I want to make sure if we really need it . So the idea behind it is able to use `Contextual API` even when we set the following in any of admin level API class
```php
/** @var bool Define if all the path building for admin api */
protected $_apiForAdmin = false;
```
This will allow us to define the api level but at the same time override if encounter a case when an API class may contains both the admin and contextual level api . But right now I can not think of such case, so this is just an enhancement placed based on future possibilities. But is it necessary or Should I remove it ?

Reply:
Make sense , removed .